src/user/model.js

```diff
@@ -1,38 +1,97 @@
-// id - integer (auto create by MySql/Postgres), PK
+// hooks :: https://sequelize.org/docs/v6/other-topics/hooks/
+// useful src :: https://github.com/sequelize/sequelize/blob/v6/src/hooks.js#L7
 
-// username - String, unique, required
-
-// hashedPassword - String, unique, required
 
+// id       - integer (auto create by MySql/Postgres), PK
```

Comment:
Please remove the following;

bio,
fullname,
email

And please revert the password field to hashedPassword

Reply:
done